### PR TITLE
Update to support commonmark 0.17

### PR DESCRIPTION
**Pull request by jdavidbakr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,11 +10,14 @@ php:
...
   - 5.6
   - 7.0
+  - 7.1
+  - 7.2
   - hhvm
 
 env:
   - COMMONMARK_VERSION=0.14.*
   - COMMONMARK_VERSION=0.15.*
+  - COMMONMARK_VERSION=0.16.*
```
Comment (GrahamCampbell) [inline .travis.yml]: We need 0.17 here too?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/AttributesBlockParser.php [hunk 2]
```diff
@@ -26,7 +26,7 @@ public function parse(ContextInterface $context, Cursor $cursor)
             return false;
         }
 
-        if (null !== $cursor->getFirstNonSpaceCharacter()) {
+        if (null !== $cursor->getNextNonSpaceCharacter()) {
```
Comment (GrahamCampbell) [inline src/AttributesBlockParser.php:29]: Looks like this is broken on 0.14?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/AttributesBlockParser.php (hunk 2 above)
Comment (jdavidbakr) [reply src/AttributesBlockParser.php:29]: Updated to remove 0.14 in the dependency tree

File: .travis.yml (hunk 1 above)
Comment (jdavidbakr) [reply .travis.yml]: Oops - missed that, it's been updated.

Comment (GrahamCampbell) [reply .travis.yml]: 👍

**Comment by GrahamCampbell** [comment]
Tests failing on 5.5.

**Review by nonoesp — APPROVED** [review]
LGTM!

**Comment by Sylry** [comment]
Looks great, tested with commonmark 0.17 and it works

**Comment by hason** [comment]
@jdavidbakr Thanks!

**Comment by nonoesp** [comment]
@jdavidbakr @hason Thanks for this!

**Pull request merged**